### PR TITLE
5804: Prints file name of invalid json file when parsing

### PR DESCRIPTION
**Pull request by renato2099**
## Description
Enhances handling runtime exception thrown when parsing JSON by adding file name to the exception message.
This was proposed in https://github.com/apache/incubator-pinot/issues/5804

## Upgrade Notes
Does this PR prevent a zero down-time upgrade? (Assume upgrade order: Controller, Broker, Server, Minion)
No

Does this PR fix a zero-downtime upgrade introduced earlier?
No

Does this PR otherwise need attention when creating release notes? 
No

### CODE REVIEW

**Comment by Jackie-Jiang** [comment]
It is better to log the bad file name in the `CreateSegmentCommand.java` under the catch of line 251, where bad files of all types can be logged

**Comment by renato2099** [comment]
I was not sure which were the code paths that could use the json parsing, that is why I added it where it actually takes place.
I moved it now to the createSegmentCommand as per your suggestion @Jackie-Jiang  I decided to keep the test as I think it is good that we add tests for these commands

File: pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/CreateSegmentCommand.java [hunk 1]
```diff
@@ -252,7 +252,8 @@ public boolean execute()
             if (j < _retry) {
               LOGGER.warn("Caught exception while creating/verifying segment, will retry", e);
             } else {
-              throw e;
+              final String exceptionMessage = String.format("%s. File: %s", e.getMessage(), dataFiles.get(sequenceId));
+              throw new Exception(exceptionMessage, e);
```
Comment (Jackie-Jiang) [inline pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/CreateSegmentCommand.java:256]:
```suggestion
              throw new RuntimeException("Caught exception while generating segment from file: " + dataFiles.get(sequenceId), e);
```

File: pinot-tools/src/test/java/org/apache/pinot/tools/TestCreateSegmentCommand.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+  private static CreateSegmentCommand _createSegmentCommand = new CreateSegmentCommand();
+
+  @Test(
+    expectedExceptions = Exception.class,
+    expectedExceptionsMessageRegExp = "^.*test_invalid_data.json.*$"
+  )
+  public void testReadingInvalidJsonFile()
+      throws Exception {
+    final String fileDirectoryPath = JSON_INVALID_SAMPLE_DATA_FILE.getParent();
```
Comment (Jackie-Jiang) [inline pinot-tools/src/test/java/org/apache/pinot/tools/TestCreateSegmentCommand.java:47]: (nit) We don't usually put `final` for local variables

File: pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/CreateSegmentCommand.java (hunk 1 above)
Comment (renato2099) [reply pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/CreateSegmentCommand.java]: updated

File: pinot-tools/src/test/java/org/apache/pinot/tools/TestCreateSegmentCommand.java (hunk 2 above)
Comment (renato2099) [reply pinot-tools/src/test/java/org/apache/pinot/tools/TestCreateSegmentCommand.java]: sure, btw is there a code style guide where I can check these things?

Comment (Jackie-Jiang) [reply pinot-tools/src/test/java/org/apache/pinot/tools/TestCreateSegmentCommand.java]: We use the style config file located at `config/codestyle-intellij.xml` (`config/codestyle-eclipse.xml` for eclipse)
Will add some code style guide in the Pinot doc

**Pull request merged**